specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2024-09-01/Watchlists.json

```diff
@@ -155,19 +155,29 @@
           }
         ],
         "responses": {
-          "200": {
-            "description": "OK"
+          "202": {
```

Comment:
This changing of API behaviour from sync to async is ok ? Since its a stable version, it will create breaking changes for sure.